ldapauthenticator/ldapauthenticator.py

```diff
@@ -455,38 +455,41 @@ def resolve_username(self, username_supplied_by_user):
             search_filter=search_filter,
             attributes=[self.lookup_dn_user_dn_attribute],
         )
-        response = conn.response
-        if len(response) == 0:
+
+        # identify unique search response entry
+        n_entries = len(conn.entries)
+        if n_entries == 0:
             self.log.warning(
                 f"Failed to lookup a DN for username '{username_supplied_by_user}'"
             )
             return (None, None)
-        if len(response) > 1:
+        if n_entries > 1:
             self.log.warning(
```

Comment:
```suggestion
            self.log.error(
```
Now that you've identified the underlying cause I think we can make this an error instead of just a warning